src/extensions/WorkspaceView/SortingInstructions.md

```diff
@@ -0,0 +1,4 @@
+## Spike sorting
```

Comment:
These are the instructions that appear when user clicks on "Spike Sorting" button.